assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -42,6 +42,7 @@ export default function SettingsView() {
 	const isGA4Enabled = useFeature( 'ga4setup' );
 	const ga4PropertyID = useSelect( ( select ) => isGA4Enabled ? select( MODULES_ANALYTICS_4 ).getPropertyID() : '' );
 	const ga4MeasurementID = useSelect( ( select ) => isGA4Enabled ? select( MODULES_ANALYTICS_4 ).getMeasurementID() : '' );
+	const webDataStreamID = useSelect( ( select ) => isGA4Enabled ? select( MODULES_ANALYTICS_4 ).getWebDataStreamID() : '' );
```

Comment:
I think this needs double checking.  For me it was opening a web data stream for a totally different property (which makes no sense given the Url includes the correct property!)